common/config.go

```diff
@@ -28,3 +26,25 @@ type Config struct {
 	// AWSURL is the URL for AWS (internal use only).
 	AWSURL string `json:"aws.url"`
 }
+
+//go:generate paramgen -output=source_paramgen.go SourceConfig
+
+type SourceConfig struct {
```

Comment:
Traditionally, I think we'd move these to separate files (config>source.go) and (config/destination.go), also having `config>config.go)

@maha-hajja what do you think?

Reply:
@raulb @maha-hajja  I moved them there so that I could use the generated keys in test utils. Otherwise I get an import cycle, as `source` and `destination` import `testutils` for making tests shorter, and `testutils` imports both back to use the configuration keys.

Now that I'm writing this I realize there's another option. We could move the `SourceConfig` and `DestinationConfig` to their respective folders, and we wouldn't have this problem anymore.

Reply:
Oh damn I forgot about the acceptance tests, they also use source and destination config keys. I have to duplicate test configuration if I move code as I said.

So I have no problem moving the source destination code into either the `config` package or the `common` package.

Reply:
I'd need to look more closely into this, but I believe the package would be `config` and imported from the source/destination package. Tests would make use of these latter packages. Something like in https://github.com/conduitio-labs/conduit-connector-oracle/pull/155/files#diff-7fd240e53ce4c101a889038fcb325fdd36d10e18e771eaa78fdbbf8ea8c700a3 I think. @maha-hajja might know better since she has more experience building connectors than I do.

Reply:
I think S3 is a good example of how to organize the configurations when you have common ones https://github.com/ConduitIO/conduit-connector-s3 (config>config.go for common configs, source>config.go for sourceConfig, and destination>config.go for destConfig), generally we'd like the source configs separated from the destination's (not in the same file) if they are different in any way